setup.py

```diff
@@ -20,7 +20,11 @@
     version=VERSION,
     packages=find_packages(include=["*"]),
     zip_safe=False,
-    install_requires=["apache-airflow>=2.0", "pandas>=1.0.0", "filelock>=3.0.0", "ray"],
+    install_requires=["apache-airflow>=2.0",
+                      "pandas>=1.0.0",
+                      "filelock>=3.0.0",
+                      "ray>=1.3.0",
```

Comment:
Let's DEFINITELY bump this one. @richardliaw @grechaw what should the minimum be?

Reply:
Can we get away with minimum 1.5.2?